scripts/create-aks-acr.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+: "${AKS_NAME:?Environment variable empty or not defined.}"
+: "${ACR_NAME:?Environment variable empty or not defined.}"
+
+register_feature() {
+  # pinning to 0.5.87 because of https://github.com/Azure/azure-cli/issues/23267
+  az extension add --name aks-preview --version 0.5.87
+  # register enable oidc preview feature
+  az feature register --namespace Microsoft.ContainerService --name EnableOIDCIssuerPreview >/dev/null
+  # https://docs.microsoft.com/en-us/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview
+  az feature register --namespace Microsoft.ContainerService --name UseCustomizedWindowsContainerRuntime >/dev/null
+  while [[ "$(az feature list --query "[?contains(name, 'Microsoft.ContainerService/EnableOIDCIssuerPreview')].{Name:name,State:properties.state}" | jq -r '.[].State')" != "Registered" ]] &&
+    [[ "$(az feature list --query "[?contains(name, 'Microsoft.ContainerService/UseCustomizedWindowsContainerRuntime')].{Name:name,State:properties.state}" | jq -r '.[].State')" != "Registered" ]]; do
+    sleep 20
+  done
```

Comment:
Does ratify require windows workloads? If not, this is not required.

Reply:
No, we don't need it, just removed it, thanks for suggestion!